packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -109,112 +108,103 @@ export default function GridItem( { categoryId, composite, icon, item } ) {
+					hidden
+					id={ `${ descriptionId }-${ index }` }
 				>
-					{ isEmpty && __( 'Empty pattern' ) }
-					{ ! isEmpty && <BlockPreview blocks={ item.blocks } /> }
-				</CompositeItem>
-				{ ariaDescriptions.map( ( ariaDescription, index ) => (
-					<div
-						key={ index }
-						hidden
-						id={ `${ descriptionId }-${ index }` }
-					>
-						{ ariaDescription }
-					</div>
-				) ) }
+					{ ariaDescription }
+				</div>
+			) ) }
+			<HStack
+				aria-hidden="true"
+				className="edit-site-patterns__footer"
+				justify="space-between"
+			>
 				<HStack
-					aria-hidden="true"
-					className="edit-site-patterns__footer"
-					justify="space-between"
+					alignment="center"
+					justify="left"
+					spacing={ 3 }
+					className="edit-site-patterns__pattern-title"
 				>
-					<HStack
-						alignment="center"
-						justify="left"
-						spacing={ 3 }
-						className="edit-site-patterns__pattern-title"
-					>
-						{ icon && (
-							<Icon
-								className="edit-site-patterns__pattern-icon"
-								icon={ itemIcon }
-							/>
-						) }
-						<Flex as="span" gap={ 0 } justify="left">
-							{ item.title }
-							{ item.type === PATTERNS && (
-								<Tooltip
-									position="top center"
-									text={ __(
-										'Theme patterns cannot be edited.'
-									) }
-								>
-									<span className="edit-site-patterns__pattern-lock-icon">
-										<Icon
-											style={ { fill: 'currentcolor' } }
-											icon={ lockSmall }
-											size={ 24 }
-										/>
-									</span>
-								</Tooltip>
-							) }
-						</Flex>
-					</HStack>
-					{ item.type === USER_PATTERNS && (
-						<DropdownMenu
-							icon={ moreHorizontal }
-							label={ __( 'Actions' ) }
-							className="edit-site-patterns__dropdown"
-							popoverProps={ { placement: 'bottom-end' } }
-							toggleProps={ {
-								className: 'edit-site-patterns__button',
-								isSmall: true,
-								describedBy: sprintf(
-									/* translators: %s: pattern name */
-									__( 'Action menu for %s pattern' ),
-									item.title
-								),
-								// The dropdown menu is not focusable using the
-								// keyboard as this would interfere with the grid's
-								// roving tab index system. Instead, keyboard users
-								// use keyboard shortcuts to trigger actions.
-								tabIndex: -1,
-							} }
-						>
-							{ () => (
-								<MenuGroup>
-									<MenuItem
-										onClick={ () =>
-											setIsDeleteDialogOpen( true )
-										}
-									>
-										{ __( 'Delete' ) }
-									</MenuItem>
-								</MenuGroup>
-							) }
-						</DropdownMenu>
+					{ icon && (
+						<Icon
+							className="edit-site-patterns__pattern-icon"
+							icon={ itemIcon }
+						/>
 					) }
+					<Flex as="span" gap={ 0 } justify="left">
+						{ item.title }
+						{ item.type === PATTERNS && (
+							<Tooltip
+								position="top center"
+								text={ __(
+									'Theme patterns cannot be edited.'
```

Comment:
I'm not sure the terminology used here is meaningful for users. What a `theme pattern` is? As far as I can tell, there's no other reference in the UI to the term `theme pattern`. The term `theme pattern` is not presented visually anywhere, nor used in any other non-visual context.